app/src/main/java/co/tinode/tindroid/CallFragment.java

```diff
@@ -450,7 +450,8 @@ private void handleCallInvite() {
                 // Send out a call invitation to the peer.
                 Map<String, Object> head = new HashMap<>();
                 head.put("mime", Tinode.VIDEO_CALL_MIME);
```

Comment:
`head.put("mime", Tinode.VIDEO_CALL_MIME);`

Is this needed? Should it be `text/x-drafty`?

Reply:
Removed.